### ISSUE TITLE
Focus should not loose while navigation using up and down arrow keys from area chart to volume chart or bar chart

### ISSUE THREAD

**Issue by arunkumarcision** [highcharts/highcharts#17467]
#### Expected behaviour
Focus should not loose while navigating using up and down arrow keys.  @karolkolodziej 

#### Actual behaviour
Issue. 
While we are navigating the chart using keyboard arrow keys (up and down) sometime focus moved from area chart to volume chart or bar chart and chart is become faded.

below is the link that team can use to understand the issue.

https://spintest5.dvcocloud.net/sites/aall/interactivechart.html

To render the issue use up and down arrow keys randomly.

#### Live demo with steps to reproduce

https://spintest5.dvcocloud.net/sites/aall/interactivechart.html
To render the issue use up and down arrow keys randomly.
#### Product version

Highstock9
#### Affected browser(s)
All browsers

**Comment by raf18seb** [comment]
Hi @arunkumarcision, thank you for reporting the issue.

Simplified jsFiddle demo: https://jsfiddle.net/BlackLabel/xhe0qjk4/
![navigator-series-accessibility](https://user-images.githubusercontent.com/23310742/177341518-a2f87225-7736-4540-aea0-f6212ea57acf.gif)

**Comment by oysteinmoseng** [comment]
Workaround: Set `mode` to `serialize` https://api.highcharts.com/highcharts/accessibility.keyboardNavigation.seriesNavigation.mode

**Comment by oysteinmoseng** [comment]
@arunkumarcision Do you have a demo that reproduces the issue with `serialize` active? I can not reproduce it here: https://jsfiddle.net/oysteinmoseng/suda6qw9/

**Comment by oysteinmoseng** [comment]
@arunkumarcision The option should be `accessibility: { keyboardNavigation: { seriesNavigation: { mode: 'serialize' } } }`, see the demo linked above - hope this helps as a workaround at least.

**Comment by arunkumarcision** [comment]
@oysteinmoseng This solution is working for the issue. Let me verify it.
